src/script/interpreter.cpp

```diff
@@ -669,7 +669,7 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 break;
 
                 case OP_NOP1:
-                case OP_NOP6: case OP_NOP7: case OP_NOP8: case OP_NOP9: case OP_NOP10:
+                case OP_NOP5: case OP_NOP6: case OP_NOP7: case OP_NOP8: case OP_NOP9: case OP_NOP10:
```

Comment:
prior to any LNHANCE changes, this NOP5 was on the prior line, so maybe put it back there?